en/administration/secure-platform.md

```diff
@@ -855,3 +855,8 @@ Centreon event logs are available in the following directories:
 
 Centreon offers to save the configuration of the platform. To do this, go to the
 [`Administration  >  Parameters  >  Backup`](./backup.html) menu.
+
```

Comment:
For me, the shortcut to the getting started chapter should not be present in the security chapter